src/app/routes/utils/regex/index.test.js

```diff
@@ -266,6 +279,34 @@ describe('onDemandRadioRegexPathsArray', () => {
   });
 });
 
+describe('onDemandTvRegexPathsArray', () => {
```

Comment:
What is the difference between this and the `describe onDemandTvPath` additions above?  They seem to be testing the same regex (`onDemandTvPath`)

Reply:
The difference is I was sane when I wrote one and crazy when I wrote the other